Packages/vcs/vcs/Canvas.py

```diff
@@ -4961,6 +4979,15 @@ def setcolormap(self, name):
         #   updateVCSsegments_flag = args[1]
         # except:
         #   updateVCSsegments_flag = 1
+        # Python < 3 DeprecationWarning ignored by default
```

Comment:
graphic methods can have colormaps as well, is that going to be triggered if I do:
```python
import vcs
b = vcs.createboxfill()
b.colormap = "white_to_red"
```